daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -184,7 +184,7 @@ class Template c => TemplateKey c k | c -> k where
     key : c -> k
 
     -- | The maintainers of the contract key.
-    maintainer : c -> [Party]
+    maintainer : k -> [Party]
```

Comment:
Can the documentation in `//ghc-lib/template-desugaring.md` get an update please or if you prefer, you can instruct me to do it.

Reply:
I updated it.